test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift

```diff
@@ -0,0 +1,74 @@
+import ClangFW
+import SwiftFW
+
+func foo() {
+  /*HERE*/
+}
+
+// REQUIRES: shell
+
+// RUN: %empty-directory(%t/Frameworks)
+// RUN: %empty-directory(%t/MyProject)
+
+// RUN: COMPILER_ARGS=( \
```

Comment:
I'm not sure how supported it is to assume you can set shell variables and have them persist across run lines.  This may work in practice, but are we sure lit will never run commands in a fresh environment?

Reply:
I'm not 100% certain this is officially supported. But several test cases rely on the behavior e.g. https://github.com/apple/swift/blob/master/test/ModuleInterface/ModuleCache/module-cache-deployment-target-irrelevant.swift.

Reply:
Okay, maybe we can follow up with people who know lit well, but no need to block this PR.